website/payments/admin.py

```diff
@@ -451,8 +451,12 @@ class BankAccountAdmin(admin.ModelAdmin):
         "valid_from",
         "valid_until",
         "signature",
+        "can_be_revoked",
+    )
+    readonly_fields = (
+        "created_at",
+        "can_be_revoked",
```

Comment:
Since this is a property I believe it will be implicitly read only

Reply:
If it's not read_only it produces an error